var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -94,16 +94,15 @@ class Sirius(CMakePackage, CudaPackage):
     depends_on('magma', when='+magma')
     depends_on('boost cxxstd=14 +filesystem', when='+boost_filesystem')
 
-    depends_on('spfft@1.0.3:', when='@6.4.0:')
-    depends_on('spfft+cuda', when='@6.4.0:+cuda')
-    depends_on('spfft+rocm', when='@6.4.0:+rocm')
+    depends_on('spfft+mpi', when='@6.4.0:')
```

Comment:
I just had a look again at the CMake config and the latest release requires 0.9.13. The next one will require 1.0.2, unless the requirement changes again.